org-roam-db.el

```diff
@@ -160,31 +138,24 @@ SQL can be either the emacsql vector representation, or a string."
   '((files
      [(file :unique :primary-key)
       (hash :not-null)
-      (meta :not-null)])
+      atime
+      mtime])
 
-    (ids
-     [(id :unique :primary-key)
+    (nodes
+     [(id :primary-key :not-null)
       (file :not-null)
-      (level :not-null)])
+      (level :not-null)
+      (pos :not-null)
+      tags
+      title
+      ref])
 
     (links
-     [(source :not-null)
+     [(file :not-null)
+      (source :not-null)
       (dest :not-null)
       (type :not-null)
```

Comment:
Is there a need for `type`? Or is it for migration?

Reply:
type is to retain support for other kinds of links (e.g. mail, cite)

Reply:
Ah. Good point. Don't use other types, so completely forgot that it might be useful outside of ID vs file links to notes.